spec/Section 3 -- Type System.md

```diff
@@ -757,7 +757,7 @@ type Person {
 }
 ```
 
-Valid operations must supply a nested field set for any field that returns an
+Valid operations must supply a selection of fields for any field that returns an
```

Comment:
Not directly related to this PR but a more precise wording would be to use `selectionSet` here as fragments are also valid, not only fields.

```suggestion
Valid operations must supply a valid sub _selection set_ for any field that returns an
```